admin/use-portable-atomic-arc

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Install dependencies to use `once_cell` & `portable_atomic_util::Arc` with critical-section
+# (cannot use `once_cell` with `--no-default-features` here due to workspace config in top-level `Cargo.toml`)
+cargo add once_cell -F alloc,race,critical-section -p rustls
```

Comment:
optional nit: I'd prefer to see long-form opts used where possible. The purpose of `-F` is less obvious than `--features` and we're not optimizing for shortest commands.

Reply:
should be done